### PR TITLE
Correction of example filenames in Digital Topology Doc

### PR DESCRIPTION
**Pull request by isivigno**
*Thanks a lot for contributing to DGtal, before submitting your PR, please fill up the description and make sure that all checkboxes are checked. Please remove these lines in your PR.*

# PR Description

Correction of example filenames in Digital Topology Doc

# Checklist

- [x] Unit-test of your feature with [Catch](http://dgtal.org/doc/stable/moduleCatch.html).
- [x] Doxygen documentation of the code completed (classes, methods, types, members...)
- [x] Documentation module page added or updated.
- [x] New entry in the [ChangeLog.md](https://github.com/DGtal-team/DGtal/blob/master/ChangeLog.md) added.
- [x] No warning raised in Debug ```cmake``` mode (otherwise, Travis C.I. will fail).
- [x] All continuous integration tests pass (Travis & appveyor)

### CODE REVIEW

**Comment by dcoeurjo** [comment]
Thanks Isabelle. Could you please add an entry in the Changelog? Thx

File: src/DGtal/topology/doc/moduleDigitalTopology.dox [hunk 1]
```diff
@@ -286,7 +286,7 @@ namespace DGtal {
   
  
  The resulting border can be visualized for instance by: 
- (see <a href="3dBorderExtraction_8cpp-example.html" >  3dBorderExtraction.cpp</a>)
+ (see <a href="3dBorderExtraction_8cpp_source.html" >  3dBorderExtraction.cpp</a>)
```
Comment (rolanddenis) [inline src/DGtal/topology/doc/moduleDigitalTopology.dox]: I think we should better use `@ref` (or `\ref`) in order to get a more reliable link (and a different rendering too), like `(see @ref 3dBorderExtraction.cpp)`

Comment (dcoeurjo) [reply src/DGtal/topology/doc/moduleDigitalTopology.dox]: agree

Comment (dcoeurjo) [reply src/DGtal/topology/doc/moduleDigitalTopology.dox]: done

**Comment by dcoeurjo** [comment]
Thanks @isivigno , merging

**Pull request merged**